src/Assetic/Extension/Twig/TwigFormulaLoader.php

```diff
@@ -34,7 +34,7 @@ public function __construct(\Twig_Environment $twig, LoggerInterface $logger = n
     public function load(ResourceInterface $resource)
     {
         try {
-            $tokens = $this->twig->tokenize($resource->getContent(), (string) $resource);
+            $tokens = $this->twig->tokenize(new \Twig_Source($resource->getContent(), (string) $resource), (string) $resource);
```

Comment:
When I follow this call to `Twig_Environment::tokenize` and then to `Twig_Lexer::tokenize` I see `$this->name = $source->getName();` and this is the name from second argument.